auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue

```diff
@@ -18,7 +18,7 @@
           </div>
         </div>
 
-        <template v-show="!anonAccount">
+        <div v-show="!anonAccount">
```

Comment:
is this intentional? template to div?
any reason?

Reply:
There was some linting errors when it was template
![image](https://user-images.githubusercontent.com/78374865/122604512-e7bc1900-d02a-11eb-9577-6f60b5356179.png)

Reply:
I reverted it back for address component flickering